coalib/bears/Bear.py

```diff
@@ -266,7 +266,9 @@ def __json__(cls):
         """
         Override JSON export of ``Bear`` object.
         """
-        _dict = get_public_members(cls)
+        # json cannot serialize properties, so drop them
```

Comment:
Basically as it is right now, it is working the way as before, if there was an instance of the object passed to JSONEncoder, we would get value from `get_public_member` and we will not filter that out. Basically what you're saying is to make json a regular method, or to modify JSONEncoder. This I think is more a separate issue. Here I only correct the current behaviour, and you want to introduce a new one.